static/styles/settings.css

```diff
@@ -1476,7 +1476,9 @@ $option_title_width: 180px;
 #id_realm_edit_topic_policy,
 #id_realm_message_content_edit_limit_seconds,
 #id_realm_delete_own_message_policy,
-#id_realm_message_content_delete_limit_seconds {
+#id_realm_message_content_delete_limit_seconds,
+#user_email_address_visibility,
+#realm_email_address_visibility {
     width: 325px;
 }
```

Comment:
Just a side comment -- we almost certainly want to look at replacing this rule with a `.settings-select` class that we apply to all of our settings select elements, so we stop having to fiddle with this CSS file whenever we add a new dropdown. Definitely a thing for a separate PR, though.

Reply:
We discussed this before in some PR a while ago. I intend to start a discussion in #design.